drivers/clock_control/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
+zephyr_include_directories_ifdef(
+	CONFIG_SOC_FAMILY_STM32
+	${ZEPHYR_BASE}/soc/arm/st_stm32/common
+	)
+
```

Comment:
There should be a way to get rid of this.

Reply:
This is actually not needed. The fact that you modified the #include directive was sufficient.
You can remove it.

Reply:
Replaced by adding include directive in soc/arm/st_stm32/CmakeLists.txt